src/redux/books/actions.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
# [OPTIONAL] Kindly try to fix linter errors rather than inline disabling, as linters help us write bug-free code